scheduling-api-http/src/integration-test/java/org/ow2/proactive/scheduling/api/GraphqlServiceIntegrationTest.java

```diff
@@ -247,10 +248,10 @@ public void testQueryJobsFilterByNames() {
     @Rollback
     @Test
     @Transactional
-    public void testQueryJobsFilterByContainsNames() {
+    public void testQueryJobsFilterByNotEqualNames() {
         addJobData(10);
 
-        Map<String, Object> queryResult = executeGraphqlQuery(String.format("{ jobs(%s:[{%s:\"*ob*\"}]) { edges { cursor node { id name } } } }",
+        Map<String, Object> queryResult = executeGraphqlQuery(String.format("{ jobs(%s:{%s:\"!fakeJob0\"}) { edges { cursor node { id name } } } }",
```

Comment:
it would be better for testing to enter a job existing in the list instead of a "fakejob"

Reply:
Ok